<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="allExecList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>361</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="activeExecList">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>361</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="launchBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="terminateBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>640</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirPathEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
